libs/angular/src/lib/modal/modal.module.ts

```diff
@@ -0,0 +1,18 @@
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { NggModalBodyComponent, NggModalComponent, NggModalFooterComponent, NggModalHeaderComponent } from './modal.component';
+
+const EXPORTED_DECLARATIONS = [
+    NggModalComponent,
+    NggModalHeaderComponent,
```

Comment:
Header, Body & Footer is just internal directives for the Modal Component right? If so, skip exporting them

Reply:
Yes, that's right. I will change that. 

As a side note, a future improvement would perhaps be to allow other content then just a text/string in the header and also have other content in the footer then just the two default buttons. I'm mentioning this because its something we have seen a need for in our project.